README.md

```diff
@@ -170,6 +170,16 @@ If you wish to make larger changes we generally recommend [forking](https://help
 
 ### Build Hets using Stack
 * [Install Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade) (use the generic Linux option if you are on Ubuntu).
+* Install build- and GUI-dependencies
+  * Ubuntu:
+    ```
+    sudo apt-get install libglib2.0-dev libcairo2-dev libpango1.0-dev libgtk2.0-dev libglade2-dev libncurses-dev
+    ```
```

Comment:
If they are really required, shouldn't those be added to debian/control's `Build-Depends:` instead of getting out of sync with it? Adding something like `mk-build-deps --install debian/control` might than be ok for flubuntu ;-).

BTW: To find out, which runtime libraries are required, on linux based envs one may use `readelf -d $binary` or otherwise on UNIX based envs `elfdump -d $binary` ...